frontend/src/components/opportunity/OpportunityDocuments.tsx

```diff
@@ -0,0 +1,94 @@
+import dayjs from "dayjs";
+import advancedFormat from "dayjs/plugin/advancedFormat";
+import timezone from "dayjs/plugin/timezone";
+
+import { useTranslations } from "next-intl";
+import { Link, Table } from "@trussworks/react-uswds";
+
+interface OpportunityDocument {
+  opportunity_attachment_type: string;
+  file_name: string;
+  download_path: string;
+  updated_at: string;
+}
+
+interface OpportunityDocumentsProps {
+  documents: OpportunityDocument[];
+}
+
+const DocumentTable = ({ documents }: OpportunityDocumentsProps) => {
+  const t = useTranslations("OpportunityListing.documents");
+
+  dayjs.extend(advancedFormat);
+  dayjs.extend(timezone);
```

Comment:
thought I made a comment here yesterday but I guess it never went through. Oops!

Anyway, can we move these plugin instantiations out of the render, and maybe even into somewhere that they can be shared / only run once? I don't know if initiating these plugins is expensive but it'd be good to limit how often it happens, since they only really need to be initiated once.